types/segment-analytics/index.d.ts

```diff
@@ -12,6 +12,7 @@ declare namespace SegmentAnalytics {
   interface SegmentOpts {
     integrations?: any;
     anonymousId?: string;
+    context?: Object;
```

Comment:
Please use the type `object` instead of `Object` as discussed [here](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).

Reply:
Updated. I also took the opportunity to replace all other occurrences of Object.

Reply:
Updated